lib/ood_core/job/adapters/slurm.rb

```diff
@@ -150,7 +151,7 @@ def squeue_required_fields
           # -u <user_list> https://slurm.schedmd.com/squeue.html
           def squeue_args(id: "", options: [])
             args  = ["--all", "--states=all", "--noconvert"]
-            args += ["-o", "#{options.join(UNIT_SEPARATOR)}"]
+            args += ["-o", "#{RECORD_SEPARATOR}#{options.join(UNIT_SEPARATOR)}"]
```

Comment:
Note that I had to move the record separator to the front to fix an error with `TRES`.